src/API.js

```diff
@@ -467,6 +470,15 @@ class Api {
 
   fetch (path, params, options) {
 
+    if (this._sdk.options.mode === 'standalone') {
+      return new Promise((resolve, reject) => {
+        if (options && options.method != "GET") reject(new Error('Can only GET in standalone mode'));
+        const a = path.split('/');
```

Comment:
This kind of assumes that a has three path parts, i dont know if thats true for all APIs. Can we make the document paths include all this:

_EXP_API= {
  '/api/experiences/blah': {}
}

as oppose to 

api.experiences.blah

Reply:
Sure I'll let Remco know so he can generate the .js json file like that.